tensorflow_datasets/core/dataset_builder.py

```diff
@@ -329,8 +362,8 @@ def download_and_prepare(self, download_dir=None, download_config=None):
       # Temporarily assign _data_dir to tmp_data_dir to avoid having to forward
       # it to every sub function.
       with utils.temporary_assignment(self, "_data_dir", tmp_data_dir):
-        if (download_config.try_download_gcs and
-            gcs_utils.is_dataset_on_gcs(self.info.full_name)):
+        if(download_config.try_download_gcs and
```

Comment:
Why did you removed the space here ? and bellow ? Could you reverse all changes which are not related to typing (e.g. in the docstring) ?

Reply:
Done